wire/wire.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto2";
```

Comment:
Is this required?

Reply:
yes, otherwise will not compile with opentracing 1.0.2